src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function `calculateRentalCost` is well-named as it clearly describes its purpose. However, the parameter `days` could be more descriptive. Consider renaming it to `numberOfDays` to indicate that it represents a count of days.